roles/satellite-clone/tasks/interface_check.yml

```
@@ -0,0 +1,44 @@
+---
+- name: Check if DNS is enabled
+  get_value_from_yaml_in_tarball:
+    tarball: "{{ backup_dir }}/config_files.tar.gz"
+    target_file: etc/foreman-installer/scenarios.d/satellite-answers.yaml
+    keys: ["foreman_proxy", "dns"]
+  register: dns_enabled
+
+- name: Check if DHCP is enabled
+  get_value_from_yaml_in_tarball:
+    tarball: "{{ backup_dir }}/config_files.tar.gz"
+    target_file: etc/foreman-installer/scenarios.d/satellite-answers.yaml
```

There is a `satellite_answers_path` variable that we can use now. It will be easier to update if things change

```suggestion
    target_file: "{{ satellite_answers_path }}"
```

---

ACK will change

---

(same goes for the other tasks with these entries)